client/my-sites/importer/logos/medium.jsx

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default () => (
+	<svg
+		/*eslint-disable wpcalypso/jsx-classname-namespace */
+		className="medium social-logo importer__service-icon"
```

Comment:
I know `medium` follows the same format as in other service icons but I'd argue that we should be more specific across the board and especially so here - `medium` as a class could mean `medium` size, or `medium` anything else.
I'd be in favour of `importer__service-icon--medium` or something like `importer__service-icon medium-service-icon`

Reply:
BTW, the changes to the other icons don't need to updated in this PR - happy for that to be tackled separately some other time :)

Reply:
That's a very good point! Especially since we also have `wix-logo`.

Reply:
Yeah! Good point - I think `medium-logo` would be fine - it's not ideal but then I don't think there's a way around it given 'medium' just is what it is :)

Reply:
I've just found the SVGs for WordPress, Blogger and Squarespace come from an external package which adds the value of the `icon` prop as a class. So even when you pass a more specific `class` prop, the element keeps the original class. You get an ugly doubling up of classes:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/1647564/56725755-62d45e00-6745-11e9-8dec-f8541c684fb5.png">

We could get around this by replacing these icons with local components like the ones for GoDaddy, Medium and Wix, but this is possibly adding a whole new bunch of complication to solve this one ambiguity. Would you be happy for me to merge this for now and have us return to the class issue another time?

Reply:
How about replacing `medium` with `medium-logo` in this component, for now, to prevent any potential collisions with other classes named `medium`?

Reply:
That's a good point - I've now pushed that change.